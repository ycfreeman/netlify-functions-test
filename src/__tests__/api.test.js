const axios = require("axios");

describe("/", () => {
  const api = axios.create({
    // baseURL: "https://knockknock.readify.net/api",
    baseURL: "http://localhost/.netlify/functions/api"
  });

  const url = "/";

  it("request", async () => {
    try {
      const { data } = await api.request({
        url
      });
      expect(data).toMatchSnapshot();
    } catch (e) {
      expect(e.response.data).toMatchSnapshot();
      expect(e).toMatchSnapshot();
    }
  });
});
