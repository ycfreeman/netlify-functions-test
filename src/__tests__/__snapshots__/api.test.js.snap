// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/ request 1`] = `""`;

exports[`/ request 2`] = `[Error: Request failed with status code 404]`;
